static/css/restyle/restyle.less

```diff
@@ -2579,12 +2579,18 @@ body.windows .install-shell .platform.mac {
   img {
     height: 1.1em;
     margin-right: 0.25em;
+    .html-rtl & {
+      margin-left: 0.25em;
```

Comment:
btw, it's the other change that is the fix for the linked issue - I just added this one too because it was a few lines above and realised it was similarly broken.